packaging/installer/netdata-updater.sh

```diff
@@ -185,8 +185,8 @@ update() {
       do_not_start="--dont-start-it"
     fi
 
-    if [ -n "${NETDATA_SELECTED_DASHBOARD}" ] ; then
-        env="NETDATA_SELECTED_DASHBOARD=${NETDATA_SELECTED_DASHBOARD}"
+    if [ -n "${NETDATA_SELECTED_DASHBOARD}" ]; then
+      env="NETDATA_SELECTED_DASHBOARD=${NETDATA_SELECTED_DASHBOARD}"
     fi
```

Comment:
Not critical for this PR, but this block can actually be removed. It's left over from when we had the switcher script for the dashboard, I just forgot to remove it when that got removed.

Reply:
Put up a PR to clean this up then :)